src/content/docs/guides/json/2-json_writing.mdx

```diff
@@ -122,6 +170,32 @@ By combining all these examples we can create the full program shown below.
   ```
 
   </TabItem>
+
+  <TabItem label="C#">
+
+  ```csharp
```

Comment:
See above comments and adjust code to ensure these are using the correct C# functions.